File: Morning Snack/JavaScript/ArmstongNumber.js
const input = require('prompt-sync')();
let userInput = input('Enter number: ');

let sum = 0;
let newUserInput = userInput;
let lengthOfNumber = newUserInput.length;
let exponential = 1;
let count;
let digit = 1

    while(newUserInput != 0){
           digit = newUserInput % 10;
           exponential = Math.pow(digit, lengthOfNumber);
           sum += exponential;
           newUserInput = Math.floor(newUserInput/10);
            console.log(newUserInput)
    }

        if(sum == userInput)
             console.log(userInput + " is an armstrong number");
        else
            console.log(userInput + " is not an armstrong number");    

File: Morning Snack/Python/armstong_number.py
user_input = int(input("Enter a number: "))
sum_of_number = 0
newuser_input = user_input

while(newuser_input != 0):
    digit = newuser_input % 10
    exponential = digit**3
    sum_of_number += exponential
    newuser_input = newuser_input//10
#print(newuser_input)


if(sum_of_number == user_input):
    print(user_input, " is an armstrong number");
else:
    print(user_input, " is not an armstrong number");    
